client/src/App.js

```diff
@@ -57,7 +57,7 @@ function App() {
           <Route path="/resourcedemo">
             <ResourcesDemo />
           </Route>
-          <Route path="/FAQ">
+          <Route path="/questions">
             <Questions />
```

Comment:
The import statement on line 19 of this file will need to be updated to import from YOUR Questions component, not Sean's hard-coded Questions component.